src/main/java/easytests/api/v1/controllers/UsersController.java

```diff
@@ -60,9 +62,33 @@ private Boolean isAdmin() {
     /**
      * show(userId)
      */
-    /**
-     * delete(userId)
-     */
+    @DeleteMapping("/{userId}")
+    public void delete(@PathVariable Integer userId) throws NotFoundException, ForbiddenException {
+        final UserModelInterface userModel = this.getUserModel(userId);
+
+        if (!this.isAdmin()) {
+            throw new ForbiddenException();
+        }
+
+        if (userModel == null) {
+            throw new NotFoundException();
+        }
+
+        final UserModelInterface userModelForDelete = this.getUserModel(userId, this.usersOptionsBuilder.forDelete());
+        this.usersService.delete(userModelForDelete);
```

Comment:
No, You should select for delete, and than delete with same option (for delete!)